modules/gitcoinPassportEligibility.json

```diff
@@ -0,0 +1,252 @@
+{
+  "name": "Gitcoin Passport Eligibility",
+  "details": [
+    "A Hats Protocol eligibility module that checks if an address has a minimum score that passes the GitcoinPassportDecoder or custom threshold."
+  ],
+  "links": [
+    {
+      "label": "GitHub",
+      "link": "https://github.com/daocoa/gitcoin-passport-eligibility/"
+    }
+  ],
+  "parameters": [
+    {
+      "label": "Gitcoin Passport Decoder",
+      "functionName": "gitcoinPassportDecoder",
+      "displayType": "default",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as an address in the module smart contract. Is this correct or should I use `token`?"
+
+    },
+    {
+      "label": "Score Criterion",
+      "functionName": "scoreCriterion",
+      "displayType": "default",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as a uint256 in the module smart contract. Is this correct or should I use `amountWithDecimals` or `hat`?"
```

Comment:
I think `default` is good here. The score criterion is a number. The `hat` display type is used when the `uint256` value represents a hat ID and then we can display that as the hat that it represents.